### PR TITLE
[12.0][MIG] sms

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
`#NothingToDo`

It's an auto_install.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by rvalyi — APPROVED** [review]

File: odoo/openupgrade/doc/source/modules110-120.rst [hunk 1]
```diff
@@ -535,7 +535,7 @@ missing in the new release are marked with |del|.
 +--------------------------------------------+-------------------------------------------------+
 |web_kanban_gauge                            | Nothing to do                                   |
 +--------------------------------------------+-------------------------------------------------+
-| |del| web_planner                          | Done. Merged in web                             |
+| |del| web_planner                          | Nothing to do: Merged in web                    |
```
Comment (pedrobaeza) [inline odoo/openupgrade/doc/source/modules110-120.rst]: This is correct as it was, as you have done something, which is merge it on `web`. Although it's the same at operational level (both ways get that the module is installed on previous version), we should be correct at semantic level.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Let the other thing untouched and it will be good to go.

File: odoo/openupgrade/doc/source/modules110-120.rst (hunk 1 above)
Comment (MiquelRForgeFlow) [reply odoo/openupgrade/doc/source/modules110-120.rst]: So, following your logic, I updated the `auth_crypt` case 😏

**Review by pedrobaeza — APPROVED** [review]
Yeah, great, the other one doesn't follow the usual semantic. Thanks for checking that.

**Comment by pedrobaeza** [comment]
You can merge when this finishes.

**Pull request merged**